references/detection/train_pytorch.py

```diff
@@ -220,7 +223,12 @@ def main(args):
     batch_transforms = Normalize(mean=(0.798, 0.785, 0.772), std=(0.264, 0.2749, 0.287))
 
     # Load doctr model
-    model = detection.__dict__[args.arch](pretrained=args.pretrained, assume_straight_pages=not args.rotation)
+    model = detection.__dict__[args.arch](
+        pretrained=args.pretrained,
+        assume_straight_pages=not args.rotation,
+        num_classes=len(val_set.class_names),
```

Comment:
`num_class` can be deduced from `class_names`, I don't think an argument `num_class` is needed. Did I miss something ?

Reply:
Yes it's possible you think it's better to remove the num_classes variable and use the info from class_names or maybe add a check to be sure it's the same because i think the num_classes variable is really important when creating a model ( always used in standard libs)

Reply:
Yes, it's better to have only one source of trust, I don't think about a situation where `num_classes` should be different from `len(class_names)`, so let's keep only one variable

Reply:
Done and pushed.